lib/generators/surveyor/install_generator.rb

```diff
@@ -1,107 +0,0 @@
-# encoding: UTF-8
-require 'rails/generators'
-
-module Surveyor
```

Comment:
We definitely need this to stay -- this pulls all the migrations over to the new app. Otherwise we can't send migrations from this gem to any installing application.